utilities/fix_password_auth.py

```diff
@@ -0,0 +1,90 @@
+import json
+import os
+import subprocess
+from optparse import OptionParser
+import sys
+
+
+def fix_password_auth(pool_file, private_network, public_network, public_network_ethernet):
+    # Check if only one of the options is set, private_network or public_network or public_network_ethernet
+    if private_network and (public_network or public_network_ethernet):
```

Comment:
This seems like code duplication, we should extract a shared method to reused with vagrant scripts